### PR TITLE
GEODE-9108: Reenable tests that fail due to no-ack region

### PR DESCRIPTION
**Pull request by mmartell**
There's 22 tests that were made to pass by simply by changing the region scope to no-ack in the server xml files. This PR reenables removes the no-ack components of the test and removes all no-ack regions from the server xml files.

### CODE REVIEW

**Comment by pdxcodemonkey** [comment]
Looks like you have a few unused local variables to delete.  Also, apparently nobody likes the `testThinClientPoolRedundancy` test, probably best to leave that one disabled.

File: cppcache/integration-test/ThinClientFailoverInterest.hpp [hunk 1]
```diff
@@ -290,7 +290,7 @@ const char* vals[] = {"Value-1", "Value-2", "Value-3", "Value-4"};
 const char* nvals[] = {"New Value-1", "New Value-2", "New Value-3",
                        "New Value-4"};
 
-const char* regionNames[] = {"DistRegionAck", "DistRegionNoAck"};
+const char* regionNames[] = {"DistRegionAck", "not-used"};
```
Comment (pdxcodemonkey) [inline cppcache/integration-test/ThinClientFailoverInterest.hpp:293]: Above, you changed this to just a static string.  Why leave it an array here?  Let's get rid of "not-used," unless of course it _is_ used, in which case "not-used" is a bad name.

File: cppcache/integration-test/ThinClientFailoverInterest2.hpp [hunk 2]
```diff
@@ -289,7 +289,7 @@ const char* vals[] = {"Value-1", "Value-2", "Value-3", "Value-4"};
 const char* nvals[] = {"New Value-1", "New Value-2", "New Value-3",
                        "New Value-4"};
 
-const char* regionNames[] = {"DistRegionAck", "DistRegionNoAck"};
+const char* regionNames[] = {"DistRegionAck", "not-used"};
```
Comment (pdxcodemonkey) [inline cppcache/integration-test/ThinClientFailoverInterest2.hpp:292]: Same

File: cppcache/integration-test/ThinClientFailoverRegex.hpp [hunk 3]
```diff
@@ -292,7 +292,7 @@ const char* vals[] = {"Value-1", "Value-2", "Value-3", "Value-4"};
 const char* nvals[] = {"New Value-1", "New Value-2", "New Value-3",
                        "New Value-4"};
 
-const char* regionNames[] = {"DistRegionAck", "DistRegionNoAck"};
+const char* regionNames[] = {"DistRegionAck", "not-used"};
```
Comment (pdxcodemonkey) [inline cppcache/integration-test/ThinClientFailoverRegex.hpp:295]: And again

File: cppcache/integration-test/ThinClientHelper.hpp [hunk 4]
```diff
@@ -134,7 +134,7 @@ const char* vals[] = {"Value-1", "Value-2", "Value-3",
 const char* nvals[] = {"New Value-1", "New Value-2", "New Value-3",
                        "New Value-4", "New Value-5", "New Value-6"};
 
-const char* regionNames[] = {"DistRegionAck", "DistRegionNoAck"};
+const char* regionNames[] = {"DistRegionAck", "not-used"};
```
Comment (pdxcodemonkey) [inline cppcache/integration-test/ThinClientHelper.hpp:137]: One more instance

File: cppcache/integration-test/testThinClientRemoteQueryTimeout.cpp [hunk 5]
```diff
@@ -290,7 +291,7 @@ DUNIT_TASK_DEFINITION(CLIENT1, StepSix)
     try {
       LOG("EXECUTE 4 START");
 
-      results = qry->execute(std::chrono::seconds(850));
+      results = qry->execute(std::chrono::seconds(850000));
```
Comment (pdxcodemonkey) [inline cppcache/integration-test/testThinClientRemoteQueryTimeout.cpp:294]: 850000 seconds is approximately 9.8 days.  Seems a little extreme as a timeout value, you probably wanted milliseconds

File: cppcache/integration-test/testThinClientRemoteQueryTimeout.cpp [hunk 6]
```diff
@@ -389,7 +390,7 @@ DUNIT_TASK_DEFINITION(CLIENT1, StepEight)
         }
       }
 
-      results = qry->execute(paramList, std::chrono::seconds(850));
+      results = qry->execute(paramList, std::chrono::seconds(850000));
```
Comment (pdxcodemonkey) [inline cppcache/integration-test/testThinClientRemoteQueryTimeout.cpp:393]: milliseconds?

File: cppcache/integration-test/testThinClientRemoteQueryTimeout.cpp [hunk 7]
```diff
@@ -447,7 +448,8 @@ DUNIT_TASK_DEFINITION(CLIENT1, verifyNegativeValueTimeout)
       LOG(logmsg.c_str());
     }
 
-    SLEEP(150000);  // sleep 2.5 min to allow server query to complete
+    // SLEEP(150000);  // sleep 2.5 min to allow server query to complete
+    SLEEP(15000);  // sleep 2.5 min to allow server query to complete
```
Comment (pdxcodemonkey) [inline cppcache/integration-test/testThinClientRemoteQueryTimeout.cpp:452]: Comment is incorrect, this is now .25 min (15 sec)

File: cppcache/integration-test/testThinClientRemoteQueryTimeout.cpp [hunk 8]
```diff
@@ -491,7 +493,8 @@ DUNIT_TASK_DEFINITION(CLIENT1, verifyLargeValueTimeout)
...
```
Comment (pdxcodemonkey) [inline cppcache/integration-test/testThinClientRemoteQueryTimeout.cpp:497]: Same

**Review by pdxcodemonkey — CHANGES_REQUESTED** [review]

File: cppcache/integration-test/ThinClientFailoverInterest.hpp (hunk 1 above)
Comment (mmartell) [reply cppcache/integration-test/ThinClientFailoverInterest.hpp:293]: Yup. Missed that one.

File: cppcache/integration-test/ThinClientFailoverInterest2.hpp (hunk 2 above)
Comment (mmartell) [reply cppcache/integration-test/ThinClientFailoverInterest2.hpp:292]: Good catch. I put in the "not-used" as a baby step, but forget to convert to scalar in a few files.

File: cppcache/integration-test/testThinClientRemoteQueryTimeout.cpp (hunk 6 above)
Comment (mmartell) [reply cppcache/integration-test/testThinClientRemoteQueryTimeout.cpp]: yup

File: cppcache/integration-test/testThinClientRemoteQueryTimeout.cpp (hunk 8 above)
Comment (mmartell) [reply cppcache/integration-test/testThinClientRemoteQueryTimeout.cpp]: Removed comment as uneccessary.

File: cppcache/integration-test/ThinClientHelper.hpp (hunk 4 above)
Comment (mmartell) [reply cppcache/integration-test/ThinClientHelper.hpp]: Lots of tests use this header. Changing to a scalar caused lots of compile issues (mostly in disabled tests). Suggest we leave this file untouched, as probably too much work to fix compile issues.

**Review by pdxcodemonkey — APPROVED** [review]

**Pull request merged**